src/kernel.cr

```diff
@@ -540,7 +540,7 @@ end
         ->Crystal::SignalChildHandler.after_fork,
 
         # reinit event loop:
-        ->Crystal::EventLoop.after_fork,
+        ->{ Crystal::Scheduler.event_loop.after_fork },
```

Comment:
Out of curiosity: why is that change needed?

Reply:
The event loop is now a class variable instead of a module. This is because the event loop implementation is detected at runtime, so it can't be a module anymore.